.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+## Purpose
```

Comment:
Are removing `issue_template.md` intentionally ?

Reply:
Yes, we will not create the issue in this repo. Hence, we don't need it.